### PR TITLE
fix: Melt UI bug & autoscroll bug on the landing page

### LINKED ISSUE #199: Remove meltui from landing page and fix auto scroll on landing page.
### Feature Description

_No response_

### Use Case

_No response_

### Proposed Solution

_No response_

### Alternatives (optional)

_No response_

### Additional Context

_No response_

### Please check the boxes that apply to this feature request.

- [ ] I have searched the existing feature requests to make sure this is not a duplicate.
- [ ] I have provided a detailed description of the requested feature.
- [ ] I have explained the use case or scenario for this feature.
- [ ] I have included any relevant technical details or design suggestions.
- [ ] I understand that this is a suggestion and that there is no guarantee of implementation.

### PR DESCRIPTION
**Pull request by tunny17**
## What does this PR do?

Fixes #199 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Chore (refactoring code, technical debt, workflow improvements)
- [ ] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change adds a new database migration
- [ ] This change requires a documentation update

## How should this be tested?

- Go to classroomio.com then click on "Our Superpowers", select any of the sub navs and study the scrolling.

### Required

- [x] Filled out the "How to test" section in this PR
- [x] Self-reviewed my own code
- [x] Commented on my code in hard-to-understand bits
- [ ] Ran `pnpm build`
- [ ] Checked for warnings, there are none
- [x] Removed all `console.logs`
- [x] Merged the latest changes from main onto my branch with `git pull origin main`
- [x] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the ClassroomIO Docs if changes were necessary

### CODE REVIEW

File: apps/classroomio-com/src/lib/Navigation/Navigation.svelte [hunk 1]
```diff
@@ -32,13 +32,29 @@
...
+          if (elAfterGoto) {
+            elAfterGoto.scrollIntoView({
+              behavior: 'smooth',
+              block: 'start',
+              inline: 'nearest'
+            });
+          }
+        }, 200);
+      }
+    }, 200);
```
Comment (rotimi-best) [inline apps/classroomio-com/src/lib/Navigation/Navigation.svelte]: @tunny17 why is there a setTimeout in a setTimeout in another setTimeout?

How does this work actually.

Note there one setTimeout in line 117, then another on line36, then another in line 46.

I think it should be something like:

```js
const scroll = () => {
  const el = document.getElementById(key);
  if (el) {
    setTimeout(() => {
      el.scrollIntoView({
          behavior: 'smooth',
          block: 'start',
          inline: 'nearest'
        });
    }, 200)
  } else {
    goto('/');
    // ... the other code here
  }
}

// even listener here
on:m-click={scroll}
```

**Review by rotimi-best — CHANGES_REQUESTED** [review]
Nice investigation, left you a feedback.

File: apps/classroomio-com/src/lib/Navigation/Navigation.svelte (hunk 1 above)
Comment (tunjiadeyemi) [reply apps/classroomio-com/src/lib/Navigation/Navigation.svelte]: lol, fixed

**Review by rotimi-best — CHANGES_REQUESTED** [review]
Please also remove uninstall `@melt-ui` and clean up any where it is imported.

I noticed in this Navigation file you updated, you didn't remove the import which is bad practice

**Review by rotimi-best — CHANGES_REQUESTED** [review]
Please uninstall melt ui from the codebase. I remember mentioning that.

It should not be in our package.json and also do a global search for any import and remove it

**Review by rotimi-best — APPROVED** [review]

**Review by rotimi-best — APPROVED** [review]
Great job, thanks for the work on this one

**Pull request merged**